Automattic/vip-go-mu-plugins
# Plugin Update: Query Monitor 3.8.2

## Description
Same as https://github.com/Automattic/vip-go-mu-plugins/pull/2805, just rebased against master for easier testing.

## Changelog Description

### Plugin Updated: Query Monitor 3.8.2

Upgraded Query Monitor from 3.7.1 to 3.8.2. Full changelog available here: https://github.com/johnbillion/query-monitor/releases (see 3.8.0 to 3.8.2).

Notably, the database query section now contains more useful information and better stack traces on VIP environments.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. View a page while logged in as a user who can view Query Monitor
1. Verify that hyperdb->query is no longer shown in the Caller column of the Queries panel in Query Monitor and that a useful calling function is shown instead
1. Verify that the query SQL and time is correctly shown for each query
1. Verify the Queries -> Query by Caller panel shows information about each caller
1. Verify that no PHP errors are triggered by Query Monitor itself
